catt/controllers.py

```diff
@@ -85,40 +94,38 @@ def get_stream(url, device_info=None, host=None, ytdl_options=None):
     return StreamInfo(url, host=host, model=cc_info, device_type=cast_type, ytdl_options=ytdl_options)
 
 
-def get_app_info(id_or_name, cast_type=None, strict=False, show_warning=False):
+def get_app(id_or_name, cast_type=None, strict=False, show_warning=False):
     if id_or_name == "default":
```

Comment:
Do we need this? If `id_or_name` is `"default"` the code below is going to return the default app anyway, no?

Reply:
True. I think I just put it there to not engage in any logic if the requester specifically needs "default".

Reply:
Sounds good to me.